Python/getargs.c

```diff
@@ -2552,7 +2552,7 @@ _PyArg_UnpackKeywordsWithVararg(PyObject *const *args, Py_ssize_t nargs,
     }
 
     /* copy keyword args using kwtuple to drive process */
-    for (i = Py_MAX((int)nargs, posonly) - varargssize; i < maxargs; i++) {
+    for (i = Py_MAX((int)nargs, posonly) - (int)varargssize; i < maxargs; i++) {
```

Comment:
```suggestion
    for (i = Py_MAX((int)nargs, posonly) - Py_SAFE_DOWNCAST(varargssize, Py_ssize_t, int); i < maxargs; i++) {
```

Reply:
Should we do this for ``nargs`` too?

Reply:
TIL about `Py_SAFE_DOWNCAST`. Thanks.

Reply:
> Should we do this for nargs too?

it probably should have done in the first place, though I wouldn't worry much for old code.